<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creategroupchat</class>
 <widget class="QDialog" name="creategroupchat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
		background-color:  qlineargradient(
		spread:pad, x1:0, y1:0 x2:0, y2:0,
		stop: 0 #A3D8F4, stop:1 #E6F7FF
		);
		border: 2px solid #ffffff;
		boder-radius: 10px;
}

QLineEdit {
		border: 2px solid #E63946;
		border-radius: 5px;
		padding: 5px;
		font-size: 14px;
		background-color: #ffffff;
		color: #333333;
}

QLineEdit:forcus {
		border: 2px solid #457B9D;
		background-color: #F1FAEE;
}

QPushButton{
		background-color: #2A9D8F;
		color: white;
		border: none;
		border-radius: 15px;
		padding: 10px 20px;
		font-size: 14px;
}

QPushButton:hover{
background-color: #264653;
}

QPushButton:pressed{
background-color: #1B3B3A;
}

QLabel{
font-size :14px;
color: #1D3557;
font-weight: bold;
}</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>71</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Group Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="roomNameInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Back  </string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/return.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="createRoomButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Create  </string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/plus.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="inviteButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Invite  </string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/add.png</normalon>
    </iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
